<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>396</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>217</width>
    <height>396</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>217</width>
    <height>396</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>201</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>在线</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>隐身</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>离线</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QTreeWidget" name="friendsListWidget">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>217</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>常规</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>高级选项</string>
     </property>
     <addaction name="showXmlAction"/>
    </widget>
    <addaction name="chatAction"/>
    <addaction name="groupChatAction"/>
    <addaction name="menu_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="addFriendAction"/>
    <addaction name="removeFriendAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="chatAction">
   <property name="text">
    <string>开始聊天</string>
   </property>
  </action>
  <action name="groupChatAction">
   <property name="text">
    <string>加入群聊</string>
   </property>
  </action>
  <action name="showXmlAction">
   <property name="text">
    <string>显示xml控制台</string>
   </property>
  </action>
  <action name="addFriendAction">
   <property name="text">
    <string>添加好友</string>
   </property>
  </action>
  <action name="removeFriendAction">
   <property name="text">
    <string>删除好友</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
